django_project/certification/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+
+from certificate import *
+from attendee import *
+from course import *
+from certifying_organisation import *
+from course_type import *
+from training_center import *
+from course_convener import *
```

Comment:
You can have this in settings, @ann26 :
View this gif, to find the settings.
![newline end](https://cloud.githubusercontent.com/assets/831865/24275553/ba42807c-1062-11e7-9d25-8d71aeca30f5.gif)